lambda/templates/create_db_users_lambda.json.tpl

```diff
@@ -8,8 +8,8 @@
         "logs:PutLogEvents"
       ],
       "Resource": [
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-checksum-${environment}",
-        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-checksum-${environment}:log-stream:*"
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-create-db-users-${environment}",
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-create-db-users-${environment}:log-stream:*"
```

Comment:
Does this fix the issue with the bastion lambda? It looks like the log group is called `tdr-create-bastion-user-intg`?

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/754712/130580997-571db5e4-9e08-4e00-a243-75de0de319f1.png">

Reply:
Grr, good spot, no its not going to fix that.